contracts/container/contract.go

```diff
@@ -255,6 +256,9 @@ func PutNamed(container []byte, signature interop.Signature,
 
 	ownerID := ownerFromBinaryContainer(container)
 	containerID := crypto.Sha256(container)
+	if storage.Get(ctx, append([]byte{deletedKeyPrefix}, []byte(containerID)...)) != nil {
+		panic(cst.ErrorDeleted)
+	}
```

Comment:
I think that we may remove the following TODO a few lines below then: `// todo: check if new container with unique container id`. It seems that it's related to the reentrance case fixed in this PR.

Reply:
Yep, fixed.